src/app/my-calc/MyEnum.ts

```diff
@@ -0,0 +1,27 @@
+export enum MathOper {
+  Multipl = "*",
+  Divide = "/",
+  Minus = "-",
+  Plus = "+",
+}
+
+
+
+
+export function checkEqual(param: string) {
+    if (param === MathOper.Plus || param === MathOper.Minus || param === MathOper.Multipl || param === MathOper.Divide) {
+        return param
+    }else{
+        return 
+    }
+    
+}
+
+export function checNotkEqual(param: string) {
+    if (param != MathOper.Plus && param != MathOper.Multipl && param != MathOper.Divide) {
```

Comment:
Kad provjeravas da nije jedanko koristi !== a ne !=

Reply:
Jesil mogo umjesto ovog ifa recimo `if(!checkEqual)` uraditi? Stim da ti i u ovom if fali Minus operacija?

Reply:
Ova not equals ti moze samo biti koliko vidim "Yes" ili undefined. Zasto onda ne korisiti boolean. Kasnije ti je lakse korisiti u drugim ifovima boolean nego explicitno pitati jeli "Yes".

Reply:
> Jesil mogo umjesto ovog ifa recimo `if(!checkEqual)` uraditi? Stim da ti i u ovom if fali Minus operacija?

mora bez minusa to je da mogu se unosit negativni brojevia da se ne mogu broju dodavat ostale operacije jer moze da ne bude error

Reply:
Ok onda moras naziv methode promjeniti, tako da je jasno sta metoda radi.